webapp-django/crashstats/crashstats/models.py

```diff
@@ -248,6 +249,10 @@ def fetch(
         retries=None,
         retry_sleeptime=None
     ):
+        valid_crash_id = find_crash_id(params['uuid'])
+        if not valid_crash_id:
+            return HttpResponseBadRequest('Invalid crash ID')
```

Comment:
This isn't going to work. This method is general across *all* the API endpoints many of which don't take a uuid parameter. We only want to validate the uuid parameter for the RawCrash API endpoint here:

https://github.com/mozilla-services/socorro/pull/4482/files#diff-0aa95732d6b7aebcee83eabf22da5dc7L696

Maybe adding validation to the `.get()` method is good? Maybe instead of putting it in the API code, put it in the model that backs it which is here:

https://github.com/mozilla-services/socorro/blob/19c0e1774ce9477cfc1c87f50926e95a340cd7ff/socorro/external/boto/crash_data.py#L50